design/src/high-level/creative-automation-case-studies.md

```diff
@@ -0,0 +1,282 @@
+# Creative Automation: Case Studies
+
+## Emergence
+
+**Conclusion:** Creativity will be low until more systems are in place, but potential is excellent. Clear and satisfying aesthetics will be key to creating a compelling contraption. Disruptions should be tunable to ensure every player can reach an optional pressure level.  
+
+### Interesting Goals
+
+Standard factory building goals are present,
+but an emphasis on sustainability and resilience add even more goals to consider.
+
+Stronger spatial and temporal variability increase the importance of considering tradeoffs and tailoring solutions to conditions.
+
+### Distinct Paths
+
+Initial prototypes will be quite weak, simply due to scale.
+
+Adding more systems and organisms will enhance this substantially.
+
+### Low Pressure
+
+Aesthetics are well-suited to chill alien cottage-core vibes.
+Disruptions are planned as an important gameplay mechanic: need excellent tuning levers to ensure that space for creativity isn't removed.
+Cascading failures shouldn't be too severe by default.
+
+### Compelling Aesthetics
+
+Smashing success.
+Unique fantasy that lends itself well to a bustling, automated hub.
+
+Aesthetic differentation will require more content than MVP.
+
+Potential for very compelling contraptions.
+Audio and timing and visual clarity will be tricky but essential to creating good contraptions.
+
+## [Factorio](https://store.steampowered.com/app/427520/Factorio/)
+
+**Conclusion:** top-tier creative automation, especially in modded games.
+Lots of options that differ in interesting ways.
+Limited aesthetic choices.
+
+### Interesting Goals
+
+Science construction is interesting and clear.
+Secondary optimization objectives are often meaningful.
+
+Trains and bots are great end products: powerful and intrinsically satisfying.
+Player upgrades like cars and suits serve as additional chase goals.
+While combat is frustrating, it serves as a clear motivator in games where it's enabled.
+
+### Distinct Paths
+
+Micro-optimizations have tons of diversity.
+Layouts, bots vs belts vs trains, modules and more.
+
+In base Factorio, there's very limited branching in recipe paths, which leads to limited choice.
+Dramatically improved in mod packs like AngelBob's.
+
+### Low Pressure
+
+Passes with flying colors.
+Biters can impact this, especially early game.
+Most people turn them off though, or keep them at a difficulty where they're not a serious challenge.
+
+Good feedback loop with biters where struggling players can simply divert resources to defense, or slow down production.
+
+### Compelling Aesthetics
+
+Great theme, compelling contraptions.
```

Comment:
Part of (but not most of) what makes the contraptions compelling is the game allow you to zoom out at a high level, and provide pretty pictures of the contraptions you make

Which is an interesting angle for such games to take, I think (perhaps even Emergence): a way to create beautiful procedurally generated art that isn't just "hurr durr random noise" --- instead it's something intricate, something meaningful, that also happens to be pretty